test_regress/t/t_assert_synth_full.pl

```diff
@@ -21,7 +21,7 @@
     check_finished => 0,
     fails => $Self->{vlt_all},
     expect =>
-'%Error: t_assert_synth.v:\d+: Assertion failed in top.t: synthesis full_case'
+'%Error: t_assert_synth.v:30: Assertion failed in top.t: synthesis full_case'
```

Comment:
If it's critical to prove the test adding the line number is ok, otherwise better to leave it general as if the test changes we then need to edit this .pl file.

Reply:
Perhaps a .out/golden file is preferrable?  Easier to update.